styles/teletype.less

```diff
@@ -416,3 +416,9 @@
   text-align: center;
   padding: 0 @component-padding @component-padding @component-padding;
 }
+
+.PackageInitializationErrorComponent {
+  color: @text-color;
+  text-align: center;
+  padding: 0 @component-padding @component-padding @component-padding;
+}
```

Comment:
Since these are the same styles used for `.PackageOutdatedComponent`, should we get rid of the duplication?

```less
.PackageOutdatedComponent, .PackageInitializationErrorComponent {
  color: @text-color;
  text-align: center;
  padding: 0 @component-padding @component-padding @component-padding;
}
```

Reply:
Good point, I will address that shortly.